chpl/chpl-service/src/main/resources/environment.properties.template

```diff
@@ -7,9 +7,15 @@ basePath=/rest
 questionableActivityEmail=theonc.certification@hhs.gov;onc_chpl@hhs.gov
 questionableActivityThresholdDays=0
 maxActivityRangeInDays=60
+
+#cache properties
 enableCacheInitialization=true
 cacheClearTimeoutSecs=15
 cacheInitializeTimeoutSecs=300
+#how long to wait before first loading the prefetched listing cache
+listingCacheRefreshInitialDelayMillis=300000

```

Comment:
Does this line mean that our "wait for the cache to be loaded" step in the tomcat/apache start scripts will have to wait an additional 300 seconds before that cache even starts loading?